Doc/library/dis.rst

```diff
@@ -487,6 +497,15 @@ operations on it as if it was a Python list. The top of the stack corresponds to
    .. versionadded:: 3.12
 
 
+.. opcode:: END_SEND
+
+   Removes the top value from the stack.
+   Equivalent to ``POP_TOP``.
+   Used to clean up when a generator exits.
```

Comment:
This is incorrect: the second item on the stack is removed, moving the top item down in its place (Equivalent to `del STACK[-2]`).

Reply:
Thanks for catching this, I missed that going over the impl.